SpotifyAPI/Local/SpotifyLocalAPI.cs

```diff
@@ -63,15 +63,23 @@ public ISynchronizeInvoke SynchronizingObject
 
         public event EventHandler<TrackTimeChangeEventArgs> OnTrackTimeChange;
 
-        public SpotifyLocalAPI(int timerIntervall = 50)
+        public SpotifyLocalAPI(int timerIntervall = 50) : this(null, timerIntervall)
         {
-            _rh = new RemoteHandler(new SpotifyLocalAPIConfig());
+        }
+
+        public SpotifyLocalAPI(ProxyConfig proxyConfig, int timerIntervall = 50)
```

Comment:
What do you think about adding a field of type `ProxyConfig` to `SpotifyLocalAPIConfig`? Would reduce our constructor overloads here and in `RemoteHandler`

Reply:
Seems like the best thing to do. I'll change it ASAP.